packages/sources/finnhub/src/endpoint/quote.ts

```diff
@@ -32,6 +32,12 @@ export const inputParameters = new InputParameters(
       description: 'The exchange to fetch data for',
       required: false,
     },
+    endpointName: {
+      type: 'string',
+      description:
+        'Is set automatically based on request endpoint alias/name. Providing a custom value has no effect',
+      required: false,
+    },
```

Comment:
Is this needed purely so that `req.requestContext` has the right type?

Reply:
As far as i remember not only for `req.requestContext` but the transport types and endpoint types as well. For example in the `builders` for websocket transport, inside the `message` handler, in input parameters and in a bunch of other places. There is no single place to force a new type as everything is connected and depend on each other. Didn't want to type cast in all those places.